internal/metastore/kv/rootcoord/suffix_snapshot.go

```diff
@@ -543,3 +553,80 @@ func (ss *SuffixSnapshot) MultiSaveAndRemoveWithPrefix(saves map[string]string,
 	}
 	return err
 }
+
+func (ss *SuffixSnapshot) Close() {
+	close(ss.closeGC)
+}
+
+// startBackgroundGC the data will clean up if key ts!=0 and expired
+func (ss *SuffixSnapshot) startBackgroundGC() {
+	log.Debug("suffix snapshot GC goroutine start!")
+	ticker := time.NewTicker(15 * time.Minute)
+	defer ticker.Stop()
+
+	params := paramtable.Get()
+	retentionDuration := params.CommonCfg.RetentionDuration.GetAsDuration(time.Second)
```

Comment:
can we just use retentionDuration?
if we set retentionDuration to 0. it may still possible that we has to use old DDL when we recover?

Reply:
Maybe retentionDuration + 1 day?

Reply:
> can we just use retentionDuration? if we set retentionDuration to 0. it may still possible that we has to use old DDL when we recover?

Yep, there will use the latest DDL to recover even though retentionDuration is 0